js/ui/html_editor/modules/toolbar.js

```diff
@@ -472,11 +473,28 @@ if(Quill) {
             return $container;
         }
 
+        _checkItemOptions(item) {
```

Comment:
maybe it's better to name this function as `warnOnRenamedOptions` or smth like this? To my mind it's not clear what we want to check now